### PR TITLE
Update .gitignore to ensure inclusion of package files

### LINKED ISSUE #5: Missing `packages/sarif-builder/tsconfig.json`
Looks like landing #1 was missing a `tsconfig.json` file that was needed for building (discovered in #4).

```
> npm run build


> sarif-js-sdk@0.0.1 build
> tsc --build

error TS5083: Cannot read file '/home/runner/work/sarif-js-sdk/sarif-js-sdk/packages/sarif-builder/tsconfig.json'.
npm ERR! code 1
npm ERR! path /home/runner/work/sarif-js-sdk/sarif-js-sdk
npm ERR! command failed
npm ERR! command sh -c tsc --build
```

### PR DESCRIPTION
**Pull request by scalvert**
Fixes #5. 

As a byproduct, it adds in missing files from the previous ignore.

### CODE REVIEW

File: packages/sarif-builder/lib/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare const _default: {};
```
Comment (rwjblue) [inline packages/sarif-builder/lib/index.d.ts:1]: packages/*/lib should be excluded (added to .gitignore)

File: packages/sarif-builder/tsconfig.tsbuildinfo [hunk 2]
```diff
@@ -0,0 +1,1211 @@
+{
```
Comment (rwjblue) [inline packages/sarif-builder/tsconfig.tsbuildinfo:1]: Should this be gitignored? I'm not sure, but I don't usually see it in repos...

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: packages/sarif-builder/lib/index.d.ts (hunk 1 above)
Comment (scalvert) [reply packages/sarif-builder/lib/index.d.ts:1]: Yep, agree. It's in .eslintignore right now, but now here. Will update.

File: packages/sarif-builder/tsconfig.tsbuildinfo (hunk 2 above)
Comment (scalvert) [reply packages/sarif-builder/tsconfig.tsbuildinfo:1]: Yes. It should.

File: packages/sarif-builder/lib/index.d.ts (hunk 1 above)
Comment (scalvert) [reply packages/sarif-builder/lib/index.d.ts:1]: Done.

File: packages/sarif-builder/tsconfig.tsbuildinfo (hunk 2 above)
Comment (scalvert) [reply packages/sarif-builder/tsconfig.tsbuildinfo:1]: Added.

Comment (rwjblue) [reply packages/sarif-builder/tsconfig.tsbuildinfo:1]: I think this still needs to be removed from the diff (its added to gitignore, but not removed from the diff)

**Review by rwjblue — APPROVED** [review]

**Pull request merged**